docker/cron.py

```diff
@@ -0,0 +1,17 @@
+import time
```

Comment:
So, we can't do normal cron jobs with docker?

Reply:
Not that I've been able to figure. One way that I contemplated was to have something that turns on the docker instance which then runs the command and goes away and there are tools that would help with this but this seemed simpler.

Reply:
What about this approach: https://stackoverflow.com/questions/37458287/how-to-run-a-cron-job-inside-a-docker-container

Reply:
Tried that. Could not get it to work. Did not feel like spinning my wheels anymore on it.

Reply:
I think we should try again, so we don't have to learn a new way of doing things. Happy to pair with you on this.